.ci/openshift-ci/images/entrypoint.sh

```diff
@@ -32,11 +37,37 @@ function terminate()
 	tail -f /dev/null
 }
 
+# Set the QEMU path on configuration.toml to $QEMU_PATH, also update
+# virtiofsd's path.
+function update_qemu_path()
```

Comment:
Nit: The `function` prefix is consistent with this file, but looks odd on this PR since other functions in other files don't use it. It's entirely redundant afaik so maybe drop it?

Reply:
done

Reply:
On that and related shell topics, any thoughts on https://github.com/kata-containers/kata-containers/pull/3100#issuecomment-976405250 ?